primefaces-selenium/primefaces-selenium-core/src/main/java/org/primefaces/selenium/internal/Guard.java

```diff
@@ -179,17 +179,25 @@ private static void waitUntilAjaxCompletes(WebDriver driver) {
             }
         }
 
-        Class<T> proxyClass = new ByteBuddy()
-                    .subclass(classToProxy)
-                    .implement(interfacesToImplement)
-                    .method(methods)
-                    .intercept(InvocationHandlerAdapter.of(handler))
-                    .make()
-                    .load(target.getClass().getClassLoader())
-                    .getLoaded();
+        proxyClass = new ByteBuddy()
+                .subclass(classToProxy)
+                .implement(interfacesToImplement)
+                .method(methods)
+                .intercept(InvocationHandlerAdapter.of(handler))
+                .make()
+                .load(target.getClass().getClassLoader())
+                .getLoaded();
 
         try {
-            return proxyClass.getDeclaredConstructor().newInstance();
+            if (classProxyable && !classToProxy.getName().contains("$ByteBuddy$") &&
+                    target instanceof WrapsElement && !(target instanceof AbstractPrimePageFragment))  {
+                // e.g. org.openqa.selenium.support.ui.Select ("native" Selenium-components?)
+                return proxyClass.getDeclaredConstructor(WebElement.class).newInstance(((WrapsElement) target).getWrappedElement());
```

Comment:
For org.openqa.selenium.support.ui.Select (and maybe some others) we need to call the constructor with WebElement-argument.
see https://github.com/SeleniumHQ/selenium/blob/9b6231a45dd4ea3bd175a3a27709bcdd107ae7e9/java/client/src/org/openqa/selenium/support/ui/Select.java#L45